src/ui/component/commentsList/view.jsx

```diff
@@ -0,0 +1,86 @@
+// @flow
+import * as React from 'react';
+import relativeDate from 'tiny-relative-date';
+
+type CommentListProps = {
+  comments: Array<any>,
+  fetchList: string => void,
+  uri: string,
+  isLoading: boolean,
+};
+
+type CommentProps = {
+  commentId: string,
+  claimId: string,
+  author: string,
+  message: string,
+  parentId: number,
+  timePosted: number,
+};
+
+class CommentList extends React.PureComponent<CommentListProps> {
+  componentDidMount() {
+    this.fetchComments(this.props);
+  }
+
+  fetchComments = (props: CommentListProps) => {
+    const { fetchList, uri } = props;
+    fetchList(uri);
+  };
+
+  render() {
+    const { comments } = this.props;
+
+    if (!comments) {
+      return null;
+    }
+
+    return (
+      <section>
+        <ul className="comments">
+          {comments.map(comment => {
+            return (
+              <Comment
+                author={comment.channel_name}
+                claimId={comment.channel_id}
+                commentId={comment.comment_id}
+                key={comment.channel_id + comment.comment_id}
+                message={comment.comment}
+                parentId={comment.parent_id || null}
+                timePosted={comment.timestamp * 1000}
+              />
+            );
+          })}
+        </ul>
+      </section>
+    );
+  }
+}
+
+class Comment extends React.PureComponent<CommentProps> {
+  render() {
+    return (
+      <li className={this.props.parentId ? 'comment reply' : 'comment'}>
+        <div className="comment__meta card__actions--between">
+          {this.props.author && <strong>{this.props.author}</strong>}
```

Comment:
` <strong>{this.props.author ? this.props.author : __("Anonymous") }</strong>`

Reply:
(note i18n call as well as simplification)

Reply:
Use this
```js
{this.props.author}
```

 in place of 
```js
  <Button className="button--uri-indicator" navigate={this.props.uri}>
          {inner}
 </Button>
  );
}
```
The schema you're using is a little out of date, you may want to compare it with the one I have on [my branch](https://github.com/osilkin98/lbry-desktop/blob/953ee3988fc6ca4af2514c9dd2a3761764feccfd/src/ui/component/uriIndicator/view.jsx#L6)